### PR TITLE
Run introspector in the host

### PR DESCRIPTION
**Pull request by ori-amizur**

### CODE REVIEW

File: internal/bminventory/inventory.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ const ignitionConfigFormat = `{
 "units": [{
 "name": "introspector.service",
 "enabled": true,
-"contents": "[Service]\nType=oneshot\nExecStart=docker run --rm --privileged --net=host quay.io/oamizur/introspector:latest /usr/bin/introspector --host %s --port %s --cluster-id %s\n\n[Install]\nWantedBy=multi-user.target"
+"contents": "[Service]\nType=simple\nExecStartPre=docker run --privileged --rm -v /usr/local/bin:/hostbin quay.io/oamizur/introspector:latest cp /usr/bin/introspector /usr/sbin/dmidecode /hostbin\nExecStart=/usr/local/bin/introspector --host %s --port %s --cluster-id %s\n\n[Install]\nWantedBy=multi-user.target"
```
Comment (filanov) [inline internal/bminventory/inventory.go:73]: you don't actually need `--privileged` right?

Comment (ori-amizur) [reply internal/bminventory/inventory.go:73]: Yes.  Seems do.

Comment (filanov) [reply internal/bminventory/inventory.go:73]: then please remove

Comment (ori-amizur) [reply internal/bminventory/inventory.go]: @filanov Doesn't work without --privileged

Comment (filanov) [reply internal/bminventory/inventory.go:74]: why not?

**Pull request merged**